lib/localeapp/cli/install.rb

```diff
@@ -118,7 +134,7 @@ def validate_key
           if key.nil?
             @output.puts "ERROR: No api key found in heroku config, have you installed the localeapp addon?"
             return
-          else
+          elsif not self.dotenv
```

Comment:
Not sure I understand the intent here, maybe we should add an
unit-test covering the change.